apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/InGax.cs

```diff
@@ -47,36 +47,31 @@ public static long CheckArgumentRange(long argument, string paramName, long minI
     public sealed class BatchingSettings
     {
         /// <summary>
-        /// Creates a new instance.
+        /// Create a new instance with the specified settings.
         /// </summary>
-        public BatchingSettings() { }
-
-        internal BatchingSettings(BatchingSettings other)
+        /// <param name="elementCountThreshold">The element count above which further processing of a batch will occur.</param>
+        /// <param name="requestByteThreshold">The byte count above which further processing of a batch will occur.</param>
+        /// <param name="delayThreshold">The batch lifetime above which further processing of a batch will occur.</param>
+        public BatchingSettings(long? elementCountThreshold, long? requestByteThreshold, TimeSpan? delayThreshold)
         {
-            ElementCountThreshold = other.ElementCountThreshold;
-            RequestByteThreshold = other.RequestByteThreshold;
-            DelayThreshold = other.DelayThreshold;
+            ElementCountThreshold = elementCountThreshold;
+            RequestByteThreshold = requestByteThreshold;
+            DelayThreshold = delayThreshold;
```

Comment:
Validation that nothing is negative? Presumably everything can be 0, meaning effectively "send immediately"? (Or maybe the bytes and element counts should be 1 or more...)